src/PullToRefresh/widget/PullToRefresh.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <widget id="PullToRefresh.widget.PullToRefresh" needsEntityContext="false"  offlineCapable="true" xmlns="http://www.mendix.com/widget/1.0/">
-    <name>Pull to refresh</name>
-    <description>Use touch on mobile enabling pull down to reload a page</description>
+    <name>Pull Action</name>
```

Comment:
Not sure if we should change the name

Reply:
Will revert, as the preview image a image anyway